bot/commands/music.py

```diff
@@ -0,0 +1,31 @@
+"""music commands for recommending music"""
+
+from discord.ext import commands
+
+from bot.backend.apis import music
+from bot.internal.bot import UtilityBot
+from bot.internal.context import UtilityContext
+
+# For debugging. Remove later
+import pprint
+
+
+class Music(commands.Cog):
+    "Standard music commands"
+
+    def __init__(self, bot: UtilityBot) -> None:
+        self.bot = bot
+
+        # Using a wrapper for Spotify API
+        self.api = self.bot.music_client
+
+    @commands.command(name="music")
+    async def music(self, ctx: UtilityContext, *, name: str) -> None:
+        """Search the Spotify API (using spotipy) for a specific song."""
```

Comment:
The docstring here has become outdated as we're now using tekore, but as the bot users don't need to know what lib we're using, I think we should just omit it. What do you think?
```suggestion
        """Search the Spotify API for a specific song."""
```